docs/preview/02-Features/02-message-handling/04-event-hubs-azure-functions.md

```diff
@@ -0,0 +1,241 @@
+﻿---
+title: "Azure EventHubs message handling for Azure Functions"
+layout: default
+---
+
+This article describes how you can use Arcus' message handler concept with Azure Functions; allowing you to more easily port your business logic from/to Azure Functions.
+
+# Azure EventHubs message handling for Azure Functions
+Our EventHubs message pump system provides a way to receive, route, and handle incoming events, but this is not needed in an Azure Functions environment.
+Today, Azure Functions acts as a message receiver meaning that the function is triggered when an event is available but does not handle message routing and handling. In this case, it acts as the message pump.
```

Comment:
I do not entirely agree with this.  Azure Functions are triggered when an event is received: ok.  But, an Azure Function allows you to handle that event / message of course.  However, it does not provide an out-of-the-box way to provide clean implementations for different ways of handling messages.
Therefore, I'd restructure the sentence like this:

```suggestion
Today, Azure Functions acts as a message receiver meaning that the function is triggered when an event is available.  Azure Functions has no out-of-the-box way to provide a clean implementation for handling different types of messages that are received.  If you want to do that, you'll need to write all plumbing code yourself.  With `Arcus.Messaging`, we provide this for you so that you can focus on writing the actual business logic.
```

Reply:
Ok! 👍